episodes/2-keras.Rmd

```diff
@@ -140,7 +140,7 @@ By using the `hue='species'` setting for the pairplot the graphs on the diagonal
 sns.pairplot(penguins, hue="species")
 ```
 
-![][pairplot]
+![][pairplot]{alt='Grid of scatter plots and histograms comparing observed values of the four physicial attributes (variables) measured in the penguins sampled. Scatter plots illustrate the distribution of values observed for each pair of variables. On the diagonal, where one physical attribute would be compared with itself, histograms are displayed that show the distribution of values observed for that attibute, coloured according to the species of the individual sampled. The pair plot shows distinct but overlapping clusters of data points representing the different species, with no pair of variables providing a clean separation of clusters on its own.'}
```

Comment:
I would use the term 'features' here, instead of variables and attributes. Or maybe it's fine to talk of physical attributes of the penguins, but then refer to 'features' instead of 'variables'.